ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh

```diff
@@ -91,6 +81,20 @@ source "${SHARED_DIR}/vsphere_context.sh"
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/govc.sh"
 
+# select a hardware version for testing
+vsphere_version=$(govc about -json | jq -r .About.Version | awk -F'.' '{print $1}')
+if [[ ${vsphere_version} -lt 8 ]]; then
+    hw_versions=(15 17 18 19)
+else
+    hw_versions=(20)
+fi
```

Comment:
How about keep the same logic like ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh.
```
hw_versions=(15 17 18 19)
if [[ ${vsphere_version} -eq 8 ]]; then
        hw_versions=(20)
fi
```

Reply:
updated.